src/commands/cmd_server.cc

```diff
@@ -657,7 +657,7 @@ class CommandTime : public Commander {
   }
 };
 
-/* HELLO [<protocol-version> [AUTH <password>] [SETNAME <name>] ] */
+/* HELLO [<protocol-version> [AUTH [<password>|<username> <password>]] [SETNAME <name>] ] */
```

Comment:
Perhaps add a comment that `username` should be always `default` otherwise it fails. And such syntax is for compatible only.

Reply:
Sure!